packages/client/src/index.js

```diff
@@ -45,6 +45,9 @@ const loadBudibase = async () => {
   // server rendered app HTML
   appStore.actions.setAppId(window["##BUDIBASE_APP_ID##"])
 
+  // Set the flag used to determine if the app is being loaded via an iframe
+  appStore.actions.setAppEmbedded(window["##BUDIBASE_APP_EMBEDDED##"] == "true")
```

Comment:
NAB: should probably use an explicit `===` here since we want to compare against the specific value `"true"`. This will function fine, but I think technically it's safer to be explicit.